content.js

```diff
@@ -651,7 +651,39 @@ class HNEnhancer {
 
             // Get the parent element of the author element and append the container as second child
             authorElement.parentElement.insertBefore(container, authorElement.parentElement.children[1]);
-            // authorElement.appendChild(container);
+
+            // Insert summarize comment link
+            const navsElement = comment.querySelector('.navs');
+            if(navsElement) {
+                navsElement.appendChild(document.createTextNode(' | '));
+
+                const summarizeChildCommentLink = document.createElement('a');
+                summarizeChildCommentLink.href = '#';
+                summarizeChildCommentLink.textContent = 'summarize child comments';
```

Comment:
rename to 'summarize thread' with alt text 'summarize all child comments in this thread'